main.go

```diff
@@ -442,6 +443,10 @@ func handlePanicWhileMounting() {
 }
 
 func main() {
+	if strings.ToLower(os.Getenv("ENABLE_VIPER_CONFIG")) == "true" {
```

Comment:
`ENABLE_VIPER_CONFIG` name seems generic, it might collide (although rare). should we make this name specific to GCSFuse? Like, `ENABLE_GCSFUSE_VIPER_CONFIG`?